subprojects/logging/src/main/java/org/gradle/internal/logging/LoggingCommandLineConverter.java

```diff
@@ -82,6 +82,9 @@ public LoggingConfiguration convert(ParsedCommandLine commandLine, LoggingConfig
             try {
                 ConsoleOutput consoleOutput = ConsoleOutput.valueOf(consoleValue);
                 loggingConfiguration.setConsoleOutput(consoleOutput);
+                if (ConsoleOutput.Plain == consoleOutput && !loggingConfiguration.isLogLevelSet()) {
```

Comment:
It kind of sucks that the logic in `PropertiesToLogLevelConfigurationConverter` kicks in first. In `PropertiesToLogLevelConfigurationConverter` the console output value is not populated yet.